code/src/components/Header.js

```diff
@@ -0,0 +1,13 @@
+import React from "react"
```

Comment:
I like this approach, I have personally made my intro page be a part of App.js, but creating it as a separate component and importing it inside App.js gives you an opportunity to later 'manipulate' it if you wish to do so. Nice!